### PR TITLE
Attempt 1 at learning concurrency

### PR DESCRIPTION
**Pull request by matthewrobinsondev**

### CODE REVIEW

File: pkg/tui/lists.go [hunk 1]
```diff
@@ -18,50 +19,53 @@ var (
...
+	IssuesList   = 0
+	ProjectsList = 1
+	EpicsList    = 2
 )
 
-func (m *model) createAssignedIssuesList(client *jira.Client, defaults list.DefaultDelegate) *model {
-	issues, _ := getAssignedIssues(client)
+func (m *model) createAssignedIssuesList(defaults list.DefaultDelegate, wg *sync.WaitGroup) *model {
+	defer wg.Done()
+	issues, _ := getAssignedIssues(m.client)
```
Comment (theoboldalex) [inline pkg/tui/lists.go]: Not going to handle the potential error returned here?

Comment (matthewrobinsondev) [reply pkg/tui/lists.go]: @theoboldalex https://youtube.com/shorts/p-hOKFr9qGY?si=T_7s78wDhPHxsyfy

Comment (matthewrobinsondev) [reply pkg/tui/lists.go]: in all seriousness probably needs a refactor, it never actually throws an error only logs it, will refactor those select functions

Comment (matthewrobinsondev) [reply pkg/tui/lists.go]: @theoboldalex updated

**Review by vaugenwake — APPROVED** [review]

**Comment by vaugenwake** [comment]
Great stuff!

**Comment by matthewrobinsondev** [comment]
Going to close as its been rewrote over.

**Pull request closed without merging**